src/styles/globals.css

```
@@ -0,0 +1,72 @@
+@import "tailwindcss";
+
+/* ---------- 컬러 색상 ---------- */
+@theme {
```

Stylelint (17.10.0)

[error] Unexpected unknown at-rule "@theme" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)